documentation/GETTING_STARTED.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Getting Started: how to prepare data for Timelapse Feature Explorer\n",
+        "\n",
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/allen-cell-animated/colorizer-data/blob/doc/getting-started-guide/documentation/GETTING_STARTED.ipynb\">\n",
+        "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+        "</a>\n",
+        "\n",
+        "The [Timelapse Feature Explorer (TFE)](https://timelapse.allencell.org) is a web-based application designed for the interactive visualization and analysis of segmented time-series microscopy data! Data needs to be processed into a specific format to be loaded into the viewer.\n",
+        "\n",
+        "In this tutorial, you'll learn how to prepare your data for the Timelapse Feature Explorer.\n",
+        "\n",
+        "## 1. Terms\n",
+        "\n",
+        "A few key terms:\n",
+        "\n",
+        "- **Dataset**: A dataset is a single time-series containing tracked objects and features. A dataset can contain up to about 16.7 million unique objects.\n",
+        "- **Raw dataset**: The raw data that you have collected or generated, before processing into the TFE format.\n",
+        "- **Collection**: An arbitrary grouping of datasets, allowing for easy comparison of multiple datasets in the viewer.\n",
+        "- **Object ID**: An ID associated with a single segmentation at a single timepoint. In the TFE-accepted format, object IDs must be sequential, starting from 0, and be unique across the whole dataset.\n",
+        "- **Track ID**: An identifier for a unique set of objects, linking their object IDs across timepoints. Generally this describes the track of one object along the time sequence.\n",
+        "\n",
+        "## 2. Prerequisites\n",
+        "\n",
+        "### Installing `colorizer-data` and tutorial dependencies\n",
+        "\n",
+        "From a command terminal, clone this repository and run the following commands to install dependencies. This will install the necessary libraries for the example scripts and the latest release of `colorizer-data`. (You may want to do this from a virtual Python environment-- see [venv](https://docs.python.org/3/library/venv.html) or [conda](https://docs.conda.io/en/latest/) for more information.)\n",
+        "\n",
+        "> **_NOTE_**: You must be on Python version 3.9 or above. The installation may fail unexpectedly on older versions of Python."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "!git clone https://github.com/allen-cell-animated/colorizer-data.git\n",
```

Comment:
This pulls from the `main` branch. If following along, you'll need to insert the following line after this command:
```
!cd colorizer-data && git checkout doc/getting-started-guide
```